includes/class-wc-install.php

```diff
@@ -624,6 +624,16 @@ private static function get_schema() {
   PRIMARY KEY (log_id),
   KEY level (level)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}woocommerce_downloadable_product_download_log (
```

Comment:
Ah, interesting.  I'll switch it to `wc_download_log` for now.  Let me know if you guys think we should adjust.  Thanks!